### PR TITLE
HextoBin.c

### PR DESCRIPTION
**Pull request by dvk65**
Another conversion. Hope this helps

### CODE REVIEW

File: conversions/HEXtoBIN.c [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#include<stdio.h>
+#include<string.h>
+int main(){
+char hex[17], bin[65] = "";
+int i = 0;
+printf("Enter hex number: ");
+gets(hex);
+for(1=0; hex[i]!='\0'; i++){
```
Comment (ashwek) [inline conversions/HEXtoBIN.c]: it should be ``i = 0``, not ``1 = 0``

File: conversions/HEXtoBIN.c [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+strcat(bin, "0111");
+break;
+case '8':
+strcat(bin, "1000");
+break;
+case '9':
+strcat(bin, "1001");
+break;
+case 'a':
+case 'A'
```
Comment (ashwek) [inline conversions/HEXtoBIN.c]: missing ``:``

File: conversions/HEXtoBIN.c [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+case 'b':
+case 'B':
+strcat(bin, "1011");
+break;
+case 'c':
+case 'C':
+strcat(bin, "1100");
+break;
+case 'd':
+case 'D'
```
Comment (ashwek) [inline conversions/HEXtoBIN.c]: missing ``:``

**Comment by ashwek** [comment]
@dvk65 thank you for contributing, but there seems to be some problem, previously accepted commits appears in your commits section too, so i would suggest you to please close this PR & create a new PR with your code.

**Comment by dvk65** [comment]
> @dvk65 thank you for contributing, but there seems to be some problem, previously accepted commits appears in your commits section too, so i would suggest you to please close this PR & create a new PR with your code.

I'll do that. Thanks!

**Pull request closed without merging**